nuxeo-features/nuxeo-platform-imaging/nuxeo-platform-imaging-core/src/test/java/org/nuxeo/ecm/platform/picture/core/test/TestRecomputePictureViews.java

```diff
@@ -85,6 +93,16 @@ public void testRecomputePictureViews() throws Exception {
         assertNotNull(pictureViews);
         assertFalse(pictureViews.isEmpty());
 
+        // check size info
+        MultiviewPicture multiviewPicture = doc.getAdapter(MultiviewPicture.class);
+        PictureView picView = multiviewPicture.getView("Medium");
+        ImageInfo imageBlobInfo = imagingService.getImageInfo(picView.getBlob());
+        ImageInfo resizedImageInfo = picView.getImageInfo();
+        assertEquals(imageBlobInfo.getHeight(), resizedImageInfo.getHeight());
+        assertEquals(imageBlobInfo.getWidth(), resizedImageInfo.getWidth());
+        assertEquals(imageBlobInfo.getWidth(), picView.getWidth());
+        assertEquals(imageBlobInfo.getHeight(), picView.getHeight());
+
```

Comment:
I would have put this test/check where it makes more sense, in `TestImagingAdapter#testMultiviewPictureAdapter` probably (where we test the `MultiviewPicture` adapter and the picture views).